### PR TITLE
build/config/rtl8721csm/script: modify ldscript for TCP performance

### PR DESCRIPTION
**Pull request by pillip8282**
modify ldscript for network performance(TCP)

### CODE REVIEW

File: build/configs/rtl8721csm/scripts/rlx8721d_img2_ns.ld [hunk 1]
```diff
@@ -143,46 +143,119 @@ SECTIONS
 		*sem_tickwait.o (.text*)
 		*sem_timedwait.o (.text*)
 		*sem_post.o (.text*)
+		/* driver */
```
Comment (sunghan-chang) [inline build/configs/rtl8721csm/scripts/rlx8721d_img2_ns.ld:146]: Could you add this in rlx8721d_img2_is.ld as well?

File: os/net/lwip/configs/nd/Kconfig [hunk 2]
```diff
@@ -3,7 +3,13 @@
...
 #
 
-if NET_IPv6_ND
+menu "ND Support"
+
+config NET_IPv6_ND
+	depends on NET_IPv6
+	default y
+	---help---
+		Build in support for IPv6 Neighbor Discovery.
```
Comment (thapav) [inline os/net/lwip/configs/nd/Kconfig:12]: Do you mean built** in support ?

File: os/net/lwip/configs/icmp6/Kconfig [hunk 3]
```diff
@@ -3,7 +3,13 @@
...
 #
 
-if NET_IPv6_ICMP
+menu "ICMP6 Support"
+
+config NET_IPv6_ICMP
+	depends on NET_IPv6
+	default y
+	---help---
+		Build in support icmp for IPv6.
```
Comment (amogh-samsung) [inline os/net/lwip/configs/icmp6/Kconfig:12]: Does this mean 
Buil**t**-in support **for** ICMP for IPv6 ?

Comment (davidfather) [reply os/net/lwip/configs/icmp6/Kconfig:12]: It's a mandatory option for IPv6 as described in this PR summary.

File: os/net/lwip/configs/nd/Kconfig (hunk 2 above)
Comment (davidfather) [reply os/net/lwip/configs/nd/Kconfig:12]: It's a mandatory option for IPv6 as described in this PR summary.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**